src/components/DistributionComparisonModal.svelte

```diff
@@ -14,14 +14,14 @@
 
   let normalized = $store.productDimensions.normalizationType === 'normalized';
   let probeType = $store.probe.type;
-  let probeKind = $store.probe.details.kind;
+  let isCategoricalProbe =
+    $store.probe.details && $store.probe.details.kind === 'categorical';
```

Comment:
would this only work with Glean metrics? how about categorical histograms?

Reply:
My understanding is we want it to be true iff it's a Legacy probe and it's categorical, no?

Reply:
Legacy probes have `details` whereas Glean metrics don't. Categorical histograms have to have `isCategoricalProbe === true`

Reply:
ah yea thanks for the clarification. could you remind me why we don't need to check for categorical glean probes here -  distribution comparison is not available for glean probes yet?

Reply:
I didn't think Glean had categorical probes. I did a quick look up at Glean dictionary and didn't find any. Oh, are they under a different name?

Reply:
I think boolean probes have categorical view in Glam, but will need to check if we even display these. 

https://github.com/mozilla/glam/blob/f0228a3947e2f8d9bdc1dd6e5bbdd0c6e7adfc8f/src/config/glean.js#L75

Reply:
hm, if that's the case GLAM doesn't support `boolean` probes